### PR TITLE
EDSC 1284 (Improved data access UI)

### PR DESCRIPTION
**Pull request by jsiarto**

### CODE REVIEW

**Comment by mreese84** [comment]
@jsiarto, the `.input-tab` elements need to be widened so that the ESI tab doesn't spill over to 3 lines.

File: app/assets/stylesheets/modules/data_access.css.scss [hunk 1]
```diff
@@ -274,12 +275,48 @@ section.data-access-next {
...
   &:first-child {
     border-top: 0px;
   }
+
+  .input-tab {
+    border: 1px solid #e3e3e3;
+    border-top-left-radius: 4px;
+    border-top-right-radius: 4px;
+    padding: 10px;
+    width: 24%;
```
Comment (mreese84) [inline app/assets/stylesheets/modules/data_access.css.scss]: I think we can set a hard pixel width here.

File: app/views/data_access/configure.html.erb [hunk 2]
```diff
@@ -78,19 +78,25 @@
...
+                    <strong><span data-bind="text: name"></span></strong>
+                    <!-- ko if: type == 'download' -->
+                    <span class="access-item-description">Download RAW data now from your browser or access script.</span>
+                    <!-- /ko -->
+                    <!-- ko if: type == 'order' -->
+                    <span class="access-item-description">Send a <strong>free</strong> order request to the provider for processing.</span>
+                    <!-- /ko -->
                     <!-- ko if: type == 'service' -->
                     <i class="fa fa-spinner fa-spin fa-fw" data-bind="visible: $parent.loadForm()"></i>
+                    <span class="access-item-description">Select custom service options to be performed. (Subsetting, reprojection, etc.)</span>
```
Comment (mreese84) [inline app/views/data_access/configure.html.erb]: Change the last part of this line to "(e.g. subsetting, etc.)"

**Comment by mreese84** [comment]
@jsiarto, are you able to fix this test? Or do you need/want assistance from @jboukes or @mightynimble?

**Comment by jsiarto** [comment]
@mreese84 I fixed the test related to the work I was doing--but the current failure is in granule_selection_spec.rb and I don't think it's a legit failure. I'd have @mightynimble or @jboukes take a look.

File: app/assets/stylesheets/modules/data_access.css.scss [hunk 3]
```diff
@@ -288,7 +288,7 @@ section.data-access-next {
     border-top-left-radius: 4px;
     border-top-right-radius: 4px;
     padding: 10px;
-    width: 24%;
+    width: 230px;
```
Comment (mreese84) [inline app/assets/stylesheets/modules/data_access.css.scss]: @jsiarto The ESI help text still spills over to 3 lines at this width for me. Any issues bumping this up to 240px?

**Pull request merged**